packages/ping/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { logger } from "@homarr/log";
+
+export const sendPingRequestAsync = async (url: string) => {
+  try {
+    return await fetch(url).then((response) => ({ statusCode: response.status }));
+  } catch (error) {
+    logger.error(error);
+    if (error instanceof Error) {
+      return { error: error.message };
+    }
+
+    if (typeof error === "string") {
+      return { error };
+    }
+
+    return { error: "Unknown error" };
```

Comment:
Maybe it would be useful to put this in an exported function since it could be used elsewhere too?